index.html

```diff
@@ -17,205 +17,228 @@
+            <li>CSS</li>
+          </ul>
+  
+          <hxr class="skils-title">
+            <img src="images/rectangle.png" alt="Diomand shape">
+            <h3 class="language">Frameworks</h3>
+          </hxr>
             <ul>
-              <li>HTML</li>
               <li>Bootstrap</li>
-              <li>Ruby</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
             </ul>
-          </hrx>    
-          <button type="button">See Project</button>
-        </hrx>
-        <hrx class="work-cards ">
-          <h2>Profesional Art Printing Data</h2>
-          <p>A daily selection of privately personalized reads; 
-            no accounts or sign-ups required. 
-            has been the industry's standard</p>
-          
-          <hrx class="tags">
+  
+          <hxr class="skils-title">
+            <img src="images/circle.png" alt="Diomand shape">
+            <h3 class="language">Skills</h3>
+          </hxr>
             <ul>
-              <li>HTML</li>
-              <li>Bootstrap</li>
-              <li>Ruby</li>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
             </ul>
-          </hrx>    
-          <button type="button">See Project</button>
-        </hrx>
       </section>
-    </section>
+    </main>
   
-    <section class="about-us">
-      <h2 class="primary-text">About me</h2>
-      <p class="description">Hello I’m a software developer! I can help you build a product, 
-        feature or website Look through some of my work and experience! 
-        If you like what you see and have a project you need coded, 
-        don’t hestiate to contact me.</p>
-        <button type="button">Get My Resume</button>
-    </section>   
-
-    <section class="skills-container skills-category">
-      <hxr class="skils-title">
-        <img src="images/diomand.png" alt="Diomand shape">
-        <h3 class="language">Language</h3>
-      </hxr>
-        <ul>
-          <li>Javascript</li>
-          <li>Ruby</li>
-          <li>HTML</li>
-          <li>CSS</li>
-        </ul>
-
-        <hxr class="skils-title">
-          <img src="images/rectangle.png" alt="Diomand shape">
-          <h3 class="language">Frameworks</h3>
-        </hxr>
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-
-        <hxr class="skils-title">
-          <img src="images/circle.png" alt="Diomand shape">
-          <h3 class="language">Skills</h3>
-        </hxr>
+    <footer> 
+      <section class="contact">
+        <p>I'm always interested in hearing about new projects,
+        so if you'd like to chat please get in touch.</p>
+        <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Kindly make sure you use the form spree service correctly. You can check the tutorial [here](https://formspree.io/html/) on the provided lesson or  you can see this video [here](https://www.youtube.com/watch?v=H_MUJ--iAcc&t=6s)